### ISSUE TITLE
Add a command line parser

### ISSUE THREAD

**Issue by nwagenbrenner** [firelab/Behave#1]
We need to be able to accept command line args or a config file. Should we use the program options method used in the WindNinja CLI? This will require boost. Is that okay?

**Comment by WChatham** [comment]
Hey Natalie,

Jason is going to show me a command line interpreter that I may be able to
use that doesn't require boost. If that doesn't work, we can look into
adding boost. I've heard horror stories, about boost, so I'll try to keep
away from it as long as possible. :)

-Bill

**Comment by nwagenbrenner** [comment]
Okay, I can also add an option in cmake to optionally compile the boost command line stuff. Let me know what you all decide. But we'll need something for the ndfd project.

**Comment by WChatham** [comment]
OK, sounds good.

**Comment by nwagenbrenner** [comment]
Added in 4969c09. The command line parser is only compiled if CLI is defined, otherwise everything should be the same as it was.

**Comment by ksshannon** [comment]
Knowing it's none of my business, I'd like to comment the absolute hardest part about building software is dependencies.  For such a trivial task, I'd recommend just writing it yourself.

**Comment by nwagenbrenner** [comment]
Yeah, that's fair. I also asked whether we might need other boost capabilities (e.g., datetime stuff, etc.) for this model in the future and it sounds like we might, so maybe boost will be needed anyway?? At any rate, we needed something soon that could handle both command line options as well as reading from a file. This seemed easiest to me (since I was doing the work and am already using boost). It's fine with me if everyone wants to remove it. I don't care how it's done as long as it works.

**Comment by wchatham** [comment]
I've removed boost and added a simple implementation for a command line parser to perform spread rate calculations. Preliminary testing indicates it is working correctly. There is no implementation for reading from a file at this time. I'll go ahead keep this issue open until it is confirmed that what is in place now sufficiently meets needs and is indeed working correctly.
